libs/shared/util-web-sockets/src/lib/use-status-web-sockets/use-status-web-sockets.ts

```diff
@@ -0,0 +1,82 @@
+import { type EnvironmentStatus, type Status } from 'qovery-typescript-axios'
+import { useParams } from 'react-router-dom'
+import { useEnvironment } from '@qovery/domains/environments/feature'
+import { type RunningState } from '@qovery/shared/enums'
+import { ServiceRunningStatus } from '@qovery/shared/interfaces'
+import { useReactQueryWsSubscription } from '@qovery/state/util-queries'
+import { queries } from '@qovery/state/util-queries'
+
+interface WSDeploymentStatus {
+  environment: EnvironmentStatus
+  stages: {
+    applications: Status[]
+    containers: Status[]
+    databases: Status[]
+    jobs: Status[]
+  }[]
+}
+
+interface WSServiceStatus {
+  environments: {
+    id: string
+    state: RunningState
+    applications: ServiceRunningStatus[]
+    containers: ServiceRunningStatus[]
+    databases: ServiceRunningStatus[]
+    jobs: ServiceRunningStatus[]
+  }[]
+}
+
+export function useStatusWebSockets() {
```

Comment:
The naming is generic, can you be more precise ? to example `useServicesStatusesWebSockets` one thing like this (same for the name of the folder and file)

Reply:
It's because it handles environments AND services statuses.
Ideally this can also handle cluster statuses for example.